xarray/tests/test_backends.py

```diff
@@ -1842,6 +1839,7 @@ def test_unsorted_index_raises(self) -> None:
         # dask first pulls items by block.
         pass
 
+    @pytest.mark.skip()
```

Comment:
Could you add a reason why?
Especially since this was working before?

Reply:
It has a `pass` in the method body. I'll move the comment up (but don't know more than that).